arangod/Aql/RegisterPlan.cpp

```diff
@@ -238,6 +238,7 @@ void RegisterPlan::after(ExecutionNode* en) {
     case ExecutionNode::SINGLETON:
     case ExecutionNode::FILTER:
     case ExecutionNode::LIMIT:
+    case ExecutionNode::MATERIALIZE:
```

Comment:
The MaterializeNode should add the output variable of the document, not the view.